challenge_one/src/agent.spec.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+} from "forta-agent";
+
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+
+import agent, { NETHERMIND_FORTA_ADDRESS } from "./agent";
+
+import { BigNumber } from "bignumber.js";
+
+var Web3 = require("web3");
+var web3 = new Web3(Web3.givenProvider);
```

Comment:
While there's no issue in using those, typically the team uses `ethers` instead of `web3` and `ethers.BigNumber` instead of `bignumber.js`.